frontend/src/store/modules/auth/state.js

```diff
@@ -1,5 +1,9 @@
 export default {
-    currentUser: null,
+    currentUser: {
+        first_name: "",
+        last_name: "",
+        avatar_url: require("@/assets/no-photo.png"),
```

Comment:
This is not ok. Please use some another icon in component if user don't have avatar yet.